scripts/popup.js

```diff
@@ -26,4 +26,96 @@ function view_all_function(){
 
 document.getElementById('save_now').onclick = save_now_function;
 document.getElementById('recent_capture').onclick = recent_capture_function;
-document.getElementById('first_capture').onclick = first_capture_function;
\ No newline at end of file
+document.getElementById('first_capture').onclick = first_capture_function;
+
+
+
+window.onload=function(){
+
+    chrome.runtime.sendMessage({message: "sendurl"}, function(response) {
+        
+        display_alexa_info(response.sent_url);
+        display_whois_info(response.sent_url);
+    });
+    
+};
+
+function display_alexa_info(url){
+    var alexa_url = 'http://data.alexa.com/data?cli=10&dat=snbamz&url=';
+		
```

Comment:
This is free api to access alexa data but as our number of users can be in huge number and this api have a throttling limit . so after sometime if the number of requests increased then it will start to throw 503 status code (Service unavailable error). so it is better if we use proper authenticated query from alexa server